server/server.lua

```diff
@@ -317,3 +317,10 @@ exports('getBlackoutState', function() return blackout end)
 exports('getTimeFreezeState', function() return freezeTime end)
 exports('getWeatherState', function() return CurrentWeather end)
 exports('getDynamicWeather', function() return Config.DynamicWeather end)
+
+exports('getTime', function()        
```

Comment:
```suggestion
exports('getTime', function()
```
Remove trailing spaces